src/main/java/com/powsybl/openloadflow/NetworkCache.java

```diff
@@ -242,6 +263,26 @@ && onShuntUpdate(shunt, attribute)) {
                         if (onSwitchUpdate(identifiable.getId(), (boolean) newValue)) {
                             done = true;
                         }
+                    } else if (identifiable.getType() == IdentifiableType.TWO_WINDINGS_TRANSFORMER) {
```

Comment:
Maybe you can refactor a little bit just to have the leg id from attribute?

Reply:
Done